packages/adapter-firebase/src/utils.ts

```diff
@@ -0,0 +1,147 @@
+import { type AppOptions, getApps, initializeApp } from "firebase-admin/app"
+import {
+  getFirestore,
+  initializeFirestore,
+  Timestamp,
+} from "firebase-admin/firestore"
```

Comment:
I think this import pattern breaks compatibility with firebase-admin v10. Not a big deal since we label the peer dependency as v11, but it might be a nice-to-have to support v10 as well?

Reply:
Since this is going to be a major bump, I would prefer we start only supporting the latest of the package. It also helps a lot with tree-shaking to decrease the size.

Based on https://firebase.google.com/support/release-notes/admin/node#version_1100_-_16_june_2022 it's also better to just drop Node 12 support.